allennlp/modules/seq2seq_encoders/__init__.py

```diff
@@ -6,17 +6,17 @@
 
 The available Seq2Seq encoders are
 
-- `"gru"` : https://pytorch.org/docs/master/nn.html#torch.nn.GRU
-- `"lstm"` : https://pytorch.org/docs/master/nn.html#torch.nn.LSTM
-- `"rnn"` : https://pytorch.org/docs/master/nn.html#torch.nn.RNN
+- `"gru"` : https://pytorch.org/docs/master/generated/torch.nn.GRU.html#torch.nn.GRU
+- `"lstm"` : https://pytorch.org/docs/master/generated/torch.nn.LSTM.html#torch.nn.LSTM
+- `"rnn"` : https://pytorch.org/docs/master/generated/torch.nn.RNN.html#torch.nn.RNN
 - `"augmented_lstm"` : allennlp.modules.augmented_lstm.AugmentedLstm
 - `"alternating_lstm"` : allennlp.modules.stacked_alternating_lstm.StackedAlternatingLstm
 - `"alternating_highway_lstm"` : allennlp.modules.stacked_alternating_lstm.StackedAlternatingLstm (GPU only)
-- `"stacked_self_attention"` : allennlp.modules.stacked_self_attention.StackedSelfAttentionEncoder
-- `"multi_head_self_attention"` : allennlp.modules.multi_head_self_attention.MultiHeadSelfAttention
-- `"pass_through"` : allennlp.modules.pass_through_encoder.PassThroughEncoder
-- `"feedforward"` : allennlp.modules.feedforward_encoder.FeedforwardEncoder
+- `"pass_through"` : allennlp.modules.seq2seq_encoders.PassThroughEncoder
+- `"feedforward"` : allennlp.modules.seq2seq_encoders.FeedForwardEncoder
 - `"pytorch_transformer"` : allennlp.modules.seq2seq_encoders.PytorchTransformer
+- `"compose"` : allennlp.modules.seq2seq_encoders.ComposeEncoder
+- `"gated-cnn-encoder"` : allennlp.momdules.seq2seq_encoders.GatedCnnEncoder
 """
```

Comment:
I think we're also missing "stacked_bidirectional_lstm"

Reply:
Thank you for catching, added in 1d6550c.